config/api.yml

```diff
@@ -3292,7 +3292,7 @@
         - :klass: VmReconfigureRequest
           :identifier: vm_reconfigure
         - :klass: VmCloudReconfigureRequest
-          :identifier: vm_cloud_reconfigure
+          :identifier: instance_resize
```

Comment:
@jrafanie @agrare I have no idea if this is the right value here, but I'm not sure what to put - do you know?  (`vm_cloud_reconfigure` doesn't exist as a feature)

Reply:
in DM, was suggesting the same one:

```
      - :name: Reconfigure Instance
        :description: Reconfigure Instance
        :feature_type: control
        :identifier: instance_resize
```

Reply:
for whatever reason, we're not consistent... we call it Reconfigure Instance but the identifier is instance_resize... even though it's service_reconfigure, vm_reconfigure, etc. for everything else.

Reply:
It is called `instance_resize` because the underlying method that is being called is `ManageIQ::Providers::CloudManager::Vm#resize`

I'm guessing they wanted something different from reconfigure because what you are able to change on a cloud instance is far less than you can change with a vm_reconfigure call on the infra side.

As for why the display name doesn't match, got me